omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,175 +51,280 @@ running::
 
-This will install the OMERO server to /usr/local/Cellar/omero, which means you
-will find the log files in :file:`/usr/local/Cellar/omero/|release|/var/log`.
-The binaries will be linked to :file:`/usr/local/bin`::
+   Run the following command to download a build of OMERO.Server::
 
-    $ which omero
-    /usr/local/bin/omero
+    $ curl https://downloads.openmicroscopy.org/omero/5.3.4/artifacts/OMERO.server-5.3.4-ice36-b69.zip > ~/Projects/Omero/server.zip
 
-Install Ice 3.6 extension for Python and OMERO python dependencies::
+   Extract the :file:`server.zip`
 
-    $ pip install -r $(brew --prefix omero53)/share/web/requirements-py27-all.txt
-    $ cd /usr/local
-    $ bash bin/omero_python_deps
+2. Once extracted, open your :file:`.bash_profile` in a text editor, 
+   such as the built-in TextEdit app::
 
-Start database server::
+    $ open -a TextEdit.app ~/.bash_profile
 
-    $ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start
+   Add an enviornment variable :envvar:OMERO_SERVER to the :file:`.bash_profile` which points
+   to the location of the OMERO executabale::
 
-Create database and user::
+    # OMERO Server distribution directory
+    export OMERO_SERVER=~/Projects/Omero/server
 
-    $ createuser -w -D -R -S db_user
-    $ createdb -E UTF8 -O db_user omero_database
-    $ psql -h localhost -U db_user -l
+   and add the OMERO executabale to the OS X :envvar:`PATH`::
 
-Set database parameters in OMERO::
+    # Add the OMERO distribution to PATH
+    export PATH=OMERO_SERVER/bin:$PATH
 
-    $ omero config set omero.db.name omero_database
-    $ omero config set omero.db.user db_user
-    $ omero config set omero.db.pass db_password
+   Using the command-line terminal, reload your :file:`.bash_profile` using::
 
-Create and run script to initialize the OMERO database::
+    $ source ~/.bash_profile
 
-    $ export ROOT_PASSWORD=${ROOT_PASSWORD:-omero}
-    $ omero db script --password $ROOT_PASSWORD -f - | psql -h localhost -U db_user omero_database
+   To ensure OMERO is correctly linked into your OS X :envvar:`PATH`, type the following in terminal and ensure
+   you get a similar output::
 
-Set up OMERO data directory::
+    $ which omero
+    /Projects/Omero/server/bin/omero
 
-    $ export OMERO_DATA_DIR=${OMERO_DATA_DIR:-~/OMERO.data}
-    $ mkdir -p $OMERO_DATA_DIR
-    $ omero config set omero.data.dir $OMERO_DATA_DIR
+3. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
 
-See the OMERO installation script :download:`step02_omero.sh <walkthrough/osx/step02_omero.sh>`
+    $ source ~/Virtual/Omero/bin/activate
 
-Development server
-^^^^^^^^^^^^^^^^^^
+4. Install Python dependencies using pip::
 
-If you wish to build OMERO.server from source for development
-purposes, using the git repository, first use Homebrew to install the
-OMERO dependencies::
+    $ pip install -r ~Omero/server/share/web/requirements-py27-all.txt
```

Comment:
And this did not find the ``requirements-py27-all.txt`` for me either

Reply:
there are I think 2 errors in the path here: first, after the tilder, a slash is missing, second, the first subdir in the home directory is Projects, not Omero.

Reply:
third, as indicated https://github.com/openmicroscopy/ome-documentation/pull/1766/files#r140455644, after unzipping I do not have any ``server`` directory, but ``OMERO.serverxxx`` directory